External/Plugins/HaXeContext/Context.cs

```diff
@@ -241,13 +241,15 @@ public void SetHaxeEnvironment(string sdkPath)
             if (currentEnv != null) path = path.Replace(currentEnv + ";", ";");
             if (neko != null) path = Regex.Replace(path, ";[^;]+neko[/\\\\]*;", ";", RegexOptions.IgnoreCase);
             path = Regex.Replace(path, ";[^;]+haxe[/\\\\]*;", ";", RegexOptions.IgnoreCase);
-            path = path.TrimStart(new char[] { ';' });
+            path = path.TrimStart(';');
             path = sdkPath + ";" + path;
-            if (neko != null) path = neko.TrimEnd(new char[] { '/', '\\' }) + ";" + path;
+            if (neko != null) path = neko.TrimEnd('/', '\\') + ";" + path;
             Environment.SetEnvironmentVariable("PATH", path);
             currentEnv = sdkPath;
 
             LoadMetadata();
+            if (GetCurrentSDKVersion().IsGreaterThanOrEquals(new SemVer("3.3.0"))) features.SpecialPostfixOperators = new[] {'!'};
+            else features.SpecialPostfixOperators = new char[0];
```

Comment:
It seems the else is not needed since it's already initialized with an empty array in ContextFeatures itself, isn't it?